src/IPropertyBinding.cls

```diff
@@ -22,10 +22,16 @@ Public Enum BindingMode
 End Enum
 
 Public Enum BindingUpdateSourceTrigger
-    Never = 0
-    OnPropertyChanged = 1 '
-    OnKeyPress = 2 '
+    'VF: added NotSetYet  and renumbered enums

```

Comment:
if logic now needs to tell "was never set" from "nope", then `NotSetYet` is warranted.

Reply:
(the numbers don't really matter, except where `Never` was in use)